test/integration2/query/select/where.spec.js

```diff
@@ -0,0 +1,903 @@
+        });
+
+        it('allows selecting columns with an array', function () {
+          return knex('accounts')
+            .where('id', '>', 1)
+            .select(['email', 'logins'])
+            .testSql(function (tester) {
+              tester(
+                'mysql',
+                'select `email`, `logins` from `accounts` where `id` > ?',
+                [1]
+              );
+              tester(
+                'pg',
+                'select "email", "logins" from "accounts" where "id" > ?',
+                [1]
+              );
+              tester(
+                'pgnative',
+                'select "email", "logins" from "accounts" where "id" > ?',
+                [1]
+              );
+              tester(
+                'pg-redshift',
+                'select "email", "logins" from "accounts" where "id" > ?',
+                [1]
+              );
+              tester(
+                'sqlite3',
+                'select `email`, `logins` from `accounts` where `id` > ?',
+                [1]
+              );
+              tester(
+                'oracledb',
+                'select "email", "logins" from "accounts" where "id" > ?',
+                [1]
+              );
+              tester(
+                'mssql',
+                'select [email], [logins] from [accounts] where [id] > ?',
+                [1]
+              );
+            });
+        });
+
+        it('allows a hash of where attrs', function () {
+          return knex('accounts')
+            .where({ id: 1 })
+            .select('*')
+            .testSql(function (tester) {
+              tester(
+                'mysql',
+                'select * from `accounts` where `id` = ?',
+                [1],
+                [
+                  {
+                    id: 1,
+                    first_name: 'Test',
+                    last_name: 'User',
+                    email: 'test1@example.com',
+                    logins: 1,
+                    balance: 0,
+                    about: 'Lorem ipsum Dolore labore incididunt enim.',
+                    created_at: TEST_TIMESTAMP,
+                    updated_at: TEST_TIMESTAMP,
+                    phone: null,
+                  },
+                ]
+              );
+              tester(
+                'pg',
+                'select * from "accounts" where "id" = ?',
+                [1],
+                [
+                  {
+                    id: '1',
+                    first_name: 'Test',
+                    last_name: 'User',
+                    email: 'test1@example.com',
+                    logins: 1,
+                    balance: 0,
+                    about: 'Lorem ipsum Dolore labore incididunt enim.',
+                    created_at: TEST_TIMESTAMP,
+                    updated_at: TEST_TIMESTAMP,
+                    phone: null,
+                  },
+                ]
+              );
+              tester(
+                'pgnative',
```

Comment:
If I remember correctly, you can pass an array as first param, so that you can cover several identical dbs at once.

would be nice to also add cockroach if it supports this

Reply:
I rework all tests in where.spec.js to avoid duplicates. It's more concise now.

Reply:
Great job!

Reply:
I fix all tests. Maybe I can disable "is not able to run two migrations in parallel when transactions are disabled", make a TODO and an issue to fix it later ? At this time, it make a lot of noise in all tests.

Reply:
yes, please! there might be existing issue for it

Reply:
I add the issue here : https://github.com/knex/knex/issues/4789 but it seems I made a mistake when I rebase/merge... I have duplicate commits xd.

Reply:
done